network-api/networkapi/fellows/urls.py

```diff
@@ -1,13 +1,15 @@
 from django.conf.urls import url
 from networkapi.fellows import views
+from django.views.generic.base import RedirectView
 
 urlpatterns = [
     url(r'^$', views.fellows_home, name='fellowships-home'),
     url(r'^directory/$',
         views.fellows_directory,
         name='fellowships-directory'),
     url(r'^support/$', views.fellows_support, name='fellowships-support'),
-    url(r'^apply/$', views.fellows_apply, name='fellowships-apply'),
+    url(r'^get-involved/$', views.fellows_get_involved, name='fellowships-get-involved'),
+    url(r'^apply/$', RedirectView.as_view(url='/fellowships/get-involved', permanent=True), name='fellowships-apply'),
```

Comment:
@alanmoo Tais already R+'ed on the content. Can you verify this redirect was set correctly?

Reply:
You should be able to set this redirect in the Wagtail admin now. If so, let's put it there so all of them are in one spot?

Reply:
ah okay, even if these pages aren't wagtail pages? if so i'll take a look

Reply:
That's why I say it *should* work. And we want to pull these into Wagtail anyway.